scripts/get-stalkerware-indicators.py

```diff
@@ -0,0 +1,60 @@
+import os
+import yaml
+import csv
+import sys
+
+# set current path to root
+os.chdir(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+# setting path
+sys.path.append(os.getcwd())
+
+import config
+
+# check if submodule is initialized
+if not os.path.exists(config.IOC_PATH):
+    print("Submodule not initialized. Initializing now...")
+    os.system("git submodule update --init --recursive")
```

Comment:
Rather than adding a submodule, do a checkout@v3, get the IOC list, and add it in. No need to pull in their whole repo